generator/src/main/java/com/scottlogic/deg/generator/commandline/CommandLineBase.java

```diff
@@ -33,6 +33,11 @@
         description = "Defines whether to overwrite/replace existing output files")
     boolean overwriteOutputFiles = false;
 
+    @CommandLine.Option(
+        names = { "--disable-schema" },
```

Comment:
Open to suggestions as to what this should be called

Reply:
- `--disable-schema-validation`?
- `--dont-validate-json-schema`?
- `--ignore-json-schema`?

I think calling out that the schema (is a JSON schema) and is used for validation could help here.

The description describes this, but might help on the CLI directly

Reply:
think I like `--disable-schema-validation`